build/rubygem-simp-cli.spec

```diff
@@ -1,3 +1,45 @@
+%{lua:
```

Comment:
simp-rake-helpers has tests to make sure the lua is correct.  Wonder how we would test that here.

Reply:
> simp-rake-helpers has tests to make sure the lua is correct. Wonder how we would test that here.

We could build an RPM using the macro, however the rake tasks need pkg:single.  Maybe an RPM query + with/without macros?

Reply:
The releng checks that verify the component version and generate the tag changelog from the RPM spec file essentially already verify the lua is correct in the sunny day scenario.  What is missing are the tests for error cases.  At this point, I think we should defer building those error case tests until we have fleshed out all the nuances of the new build process.